cluster/ci/monitoring/mixins/grafana_dashboards/e2e_template_jobs.jsonnet

```diff
@@ -0,0 +1,112 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+
+local dashboardConfig = {
+  uid: '68542739-4831-43e0-b5a0-5117369c83c8',
```

Comment:
Can we keep it in this file?

https://github.com/openshift/release/blob/358af62b66c02d71c8bba3761e9972cf10edb3ad/cluster/ci/monitoring/mixins/config.libsonnet#L4-L7

Reply:
Can we also remove the hyphens.

@stevekuznetsov got a smart idea to obtain the UUID. `git hash`.
The command I used to do

```
$ dbus-uuidgen 
2092ae2584faf9be8c0782705d52c43d

```

Reply:
And also refer the ID here, eg,

https://github.com/openshift/release/blob/f776acc06ab821e2e842bf530793a9a39a9301ee/cluster/ci/monitoring/mixins/grafana_dashboards/dptp.jsonnet#L14-L16

`uid: config._config.grafanaDashboardIDs['e2e_template_jobs.json'], `

Reply:
we can't have duplicated `uid` field.